packages/jest-config/src/__tests__/normalize.test.js

```diff
@@ -922,7 +924,10 @@ describe('preset', () => {
     expect(options.setupFiles.sort()).toEqual([
       '/node_modules/a',
       '/node_modules/b',
-      '/node_modules/regenerator-runtime/runtime',
+    ]);
```

Comment:
`babel-jest` isn't explicitly set, so it doesn't append `regenerator-runtime` anymore.